### ISSUE TITLE
Trying to donate caused immediate crash

### ISSUE THREAD

**Issue by xnaas** [ccrama/Slide#192]
## Info
Can't donate. Only crashes.
## Stacktrace
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void me.ccrama.redditslide.util.IabHelper.flagEndAsync()' on a null object reference
at me.ccrama.redditslide.Activities.Don
ateView$2.onClick(Don
ateView.java:82)
at android.view.View.performClick(View.java:4780)
at android.view.View$PerformClick.run(View.java:19866)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.disp
atchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5254)
at java.lang.reflect.Method.invoke(N
ative Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```
Reported via Slide Crash Capture

**Comment by l3d00m** [comment]
Do you use a self compiled version? If true, that's no wonder because the secret keys that are needed for donating (/in app purchase) aren't published on github.

**Comment by xnaas** [comment]
Nope. I use the Play Store version of the Alpha and the Pro Key was bought via the Play Store.

**Comment by ccrama** [comment]
Has been fixed on my end! Sorry bout that and thanks for donating <3. Update coming in a few hours

**Comment by ccrama** [comment]
Fixed with e5cbcea7f841b658d6b45480ffa4e6a77e36e3de
